### PR TITLE
frontend: Upgrade to new iconify 3.1.3

### PR DESCRIPTION
**Pull request by joaquimrocha**
Also move to a string-based icon description.

### CODE REVIEW

File: frontend/src/components/App/__snapshots__/TopBar.stories.storyshot [hunk 1]
```diff
@@ -56,20 +56,7 @@ exports[`Storyshots TopBar No Token 1`] = `
         <span
           class="MuiIconButton-label"
         >
-          <svg
```
Comment (illume) [inline frontend/src/components/App/__snapshots__/TopBar.stories.storyshot:59]: Looks like there's no SVG anymore, and just a span instead.

**Comment by joaquimrocha** [comment]
@illume , I would like to merge this but we can eventually break existing plugins because of this required change:
```javascript
import Icon from '@iconify/react';
import { Icon } from '@iconify/react';
```

Do you think that we can do any magic on the headlamp-plugin level to still support the legacy default import? Or some other solution?

**Comment by illume** [comment]
Yeah I think we can do some sort of fixer for that in a `headlamp-plugin update` style thing. Added it to the migration issue: https://github.com/kinvolk/headlamp/issues/426#issuecomment-1058714643

I tested this with latest main, and it's ok. The storyshots issue isn't important IMHO, and I hope it will be fixed with other dependency updates coming up (if it even really needs fixing at all).

**Review by illume — APPROVED** [review]
🎉🎈

**Pull request merged**